### PR TITLE
add example code

### PR DESCRIPTION
**Pull request by hdhono**
下記の追加を行いました。確認お願いします。
train.py : 前回のトレーニングのスクリプトに保存部分を追加
evaluation.py : 保存したモデルの読み込み・テストデータで精度を評価

### CODE REVIEW

File: codes/08/example/evaluation.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# モデルの定義
+model = CNN()
+
+# 保存されたモデルの読み込み
+checkpoint = torch.load(model_path)
+model.load_state_dict(checkpoint['model_state_dict'])
+model.eval()
+
+# 損失関数の定義
+criterion = nn.CrossEntropyLoss()
```
Comment (shun74) [inline codes/08/example/evaluation.py]: evaluation.pyは精度の評価だけでいいので`loss`を計算している部分などは消してください。

**Review by shun74 — CHANGES_REQUESTED** [review]

**Review by Kosuke-Sugimoto — APPROVED** [review]
LGTM

File: codes/08/example/evaluation.py (hunk 1 above)
Comment (Kosuke-Sugimoto) [reply codes/08/example/evaluation.py]: @hdhono 
accuracyの計算にcriterionは使わないので定義から消しちゃってよい気がします
また、それに伴い loss = criterion(~) の部分も消しちゃってよいかと

**Comment by shun74** [comment]
@hdhono 
スライドを更新したので修正したものをプッシュしてください。

**Review by shun74 — APPROVED** [review]
LGTM
